src/aatoolbox/datasources/chirps/chirps.py

```diff
@@ -0,0 +1,572 @@
+"""Class to download and load CHIRPS observational precipitation data."""
+import calendar
+import logging
+from abc import abstractmethod
+from datetime import date, datetime
+from pathlib import Path
+from typing import Optional
+
+import cftime
```

Comment:
Oops should have noticed this earlier, but cftime needs to be added to the list of dependencies in `setup.py`, and the requirements files updated.

Reply:
Ah so sorry, I meant setup.cfg